tests/kubectl_test.go

```diff
@@ -90,137 +76,6 @@ var _ = Describe("[sig-compute]oc/kubectl integration", decorators.SigCompute, f
-			Entry("[test_id:3466]virtualmachineinstance", "get", "vmi", "wide", []string{"NAME", "AGE", "PHASE", "IP", "NODENAME", "READY", "LIVE-MIGRATABLE", "PAUSED"}, 1, "True"),
-		)
-
-	})
-
-	Describe("VM instance migration", decorators.RequiresTwoSchedulableNodes, func() {
-		var virtClient kubecli.KubevirtClient
-
-		BeforeEach(func() {
-			virtClient = kubevirt.Client()
-		})
-
-		Context("'kubectl get vmim'", func() {
-			It("print the expected columns and their corresponding values", func() {
-				vmi := libvmifact.NewCirros(
-					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
-					libvmi.WithNetwork(v1.DefaultPodNetwork()),
-				)
-				By("Starting the VirtualMachineInstance")
-				vmi = libvmops.RunVMIAndExpectLaunch(vmi, libmigration.MigrationWaitTime)
-
-				By("Checking that the VirtualMachineInstance console has expected output")
-				Expect(console.LoginToCirros(vmi)).To(Succeed())
-
-				By("creating the migration")
-				migration := libmigration.New(vmi.Name, vmi.Namespace)
-
-				var migrationCreated *v1.VirtualMachineInstanceMigration
-				By("starting migration")
-				Eventually(func() error {
-					migrationCreated, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(context.Background(), migration, metav1.CreateOptions{})
-					return err
-				}, libmigration.MigrationWaitTime, 1*time.Second).Should(Succeed(), "migration creation should succeed")
-				migration = migrationCreated
-
-				libmigration.ExpectMigrationToSucceedWithDefaultTimeout(virtClient, migration)
-
-				k8sClient := clientcmd.GetK8sCmdClient()
-				result, _, err := clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "get", "vmim", migration.Name)
-				// due to issue of kubectl that sometimes doesn't show CRDs on the first try, retry the same command
-				if err != nil {
-					result, _, err = clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "get", "vmim", migration.Name)
-				}
-
-				expectedHeader := []string{"NAME", "PHASE", "VMI"}
-				Expect(err).ToNot(HaveOccurred())
-				Expect(result).ToNot(BeEmpty())
-				resultFields := strings.Fields(result)
-
-				By("Verify that only Header is not present")
-				Expect(len(resultFields)).Should(BeNumerically(">", len(expectedHeader)))
-
-				columnHeaders := resultFields[:len(expectedHeader)]
-				By("Verify the generated header is same as expected")
-				Expect(columnHeaders).To(Equal(expectedHeader))
-
-				By("Verify VMIM name")
-				Expect(resultFields[len(expectedHeader)]).To(Equal(migration.Name), "should match VMIM object name")
-				By("Verify VMIM phase")
-				Expect(resultFields[len(expectedHeader)+1]).To(Equal(string(v1.MigrationSucceeded)), "should have successful state")
-				By("Verify VMI name related to the VMIM")
-				Expect(resultFields[len(expectedHeader)+2]).To(Equal(vmi.Name), "should match the VMI name")
```

Comment:
How do we these? The json path?

Reply:
The added unit test asserts the exact order of additional printer columns. Meaning that after `Name` the `Phase` needs to come first and then the `VMI`.

Reply:
That is one thing but how do we make sure we write correct json path?

Reply:
We don't check the json path that was defined for additional printer columns. This was the only test that did this. I'm not sure if it's worth the effort, but if you insist, perhaps a unit test could be written for it.

Reply:
Well either we have a test or we can verify it manually on each PR. Regressions can happen in many ways. Can we leave this test for now and merge the rest?

Reply:
@xpivarc Added tests in https://github.com/kubevirt/kubevirt/pull/14554/commits/59a53b12ca9a8828579020c0ae71b3d2c639eb20 to address your concern and fixed a regression I found in 9a292f2e968eaae6e7779032a6d9e3b1ce4bf3cb. PTAL.